### ISSUE TITLE
Stuttering nas traduções da janela de quest

### ISSUE THREAD

**Issue by ricardoaperalta** [lokinmodar/Echoglossian#109]
Gostaria de relatar que o stuttering ainda ocorre, porem agora ele ocorre se vc usa a opção de tradução das quests. Quando vc alterna entre as quest para cada troca há stutterings.

**Comment by lokinmodar** [comment]
Sim. Estamos trabalhando pra remover esse freeze por completo em todas as circunstâncias

**Comment by lokinmodar** [comment]
Closing this as resolved in the 4.x release line.

Quest-family translation/runtime behavior was reworked in 4.x specifically to remove the older stuttering/freeze behavior around quest updates and switching.

If quest-window stuttering is still reproducible on 4.x, please open a new issue and mention the exact addon/window plus translation mode.
